src/App/Application.cpp

```diff
@@ -302,12 +302,12 @@ void Application::setupPythonTypes()
     // NOTE: To finish the initialization of our own type objects we must
     // call PyType_Ready, otherwise we run into a segmentation fault, later on.
     // This function is responsible for adding inherited slots from a type's base class.
-    Base::Interpreter().addType(&Base::VectorPy::Type, pAppModule, "Vector");
-    Base::Interpreter().addType(&Base::MatrixPy::Type, pAppModule, "Matrix");
-    Base::Interpreter().addType(&Base::BoundBoxPy::Type, pAppModule, "BoundBox");
-    Base::Interpreter().addType(&Base::PlacementPy::Type, pAppModule, "Placement");
-    Base::Interpreter().addType(&Base::RotationPy::Type, pAppModule, "Rotation");
-    Base::Interpreter().addType(&Base::AxisPy::Type, pAppModule, "Axis");
+    InterpreterSingleton::addType(&Base::VectorPy::Type, pAppModule, "Vector");

```

Comment:
It’s inconsistent on Base usage. Some singletons uses base while others doesn’t. On these lines base is also both implicit and explicit.

Reply:
This was a merge error on my part, I'll fix it. Thanks for the catch!